### PR TITLE
Update protobuf blog to match new content in example repo

### PR DESCRIPTION
**Pull request by cgrushko**
: https://github.com/cgrushko/proto_library/

### CODE REVIEW

File: _posts/2017-02-27-protocol-buffers.md [hunk 1]
```diff
@@ -36,37 +36,32 @@ Bazel's proto rules implicitly depend on the
 distribution (described below, in "Implicit Dependencies and Proto Toolchains").
 The following satisfies these dependencies:
 
-> TIP: This is a shortened version of
-> [https://github.com/cgrushko/proto_library/blob/master/WORKSPACE](https://github.com/cgrushko/proto_library/blob/master/WORKSPACE)
+> TIP: Clone https://github.com/cgrushko/proto_library/ to try protobufs in Bazel now.
```
Comment (jin) [inline _posts/2017-02-27-protocol-buffers.md]: Let's linkify the github url

File: _posts/2017-02-27-protocol-buffers.md [hunk 2]
```diff
@@ -36,37 +36,32 @@ Bazel's proto rules implicitly depend on the
...
+    strip_prefix = "protobuf-5e8916e881c573c5d83980197a6f783c132d4276",
+    urls = ["https://github.com/google/protobuf/archive/5e8916e881c573c5d83980197a6f783c132d4276.zip"],
 )
 ```
 
 ### BUILD files
 
-> TIP: This is a shortened version of
-> [https://github.com/cgrushko/proto_library/blob/master/src/BUILD](https://github.com/cgrushko/proto_library/blob/master/src/BUILD)
+> TIP: Clone https://github.com/cgrushko/proto_library/ to try protobufs in Bazel now.
```
Comment (jin) [inline _posts/2017-02-27-protocol-buffers.md]: Linkify this link?

File: _posts/2017-02-27-protocol-buffers.md [hunk 3]
```diff
@@ -148,10 +143,7 @@ them independent of Bazel.
 
 **Q:** How does one use well-known types? (e.g., `any.proto`,
 `descriptor.proto`)<br />
-**A:** Once [https://github.com/google/protobuf/issues/2763](https://github.com/google/protobuf/issues/2763) is resolved, the
-following should be added to a `.proto` file: `import google/protobuf/any.proto`
-and the following: `@com_google_protobuf//:well_known_types_protos` to one's
-`proto_library` rule.
+**A:** See [the example repo](https://github.com/cgrushko/proto_library), particularly https://github.com/cgrushko/proto_library/blob/beae7b78b85b3af51d3ea54663c421ebde97dc10/src/BUILD#L42.
```
Comment (jin) [inline _posts/2017-02-27-protocol-buffers.md]: Same here, linkify the github url

File: _posts/2017-02-27-protocol-buffers.md [hunk 4]
```diff
@@ -36,37 +36,32 @@ Bazel's proto rules implicitly depend on the
...
     name = "com_google_protobuf",
-    urls = ["https://github.com/google/protobuf/archive/b4b0e304be5a68de3d0ee1af9b286f958750f5e4.zip"],
+    sha256 = "cef7f1b5a7c5fba672bec2a319246e8feba471f04dcebfe362d55930ee7c1c30",
+    strip_prefix = "protobuf-3.5.0",
+    urls = ["https://github.com/google/protobuf/archive/v3.5.0.zip"],
 )
 
-# cc_proto_library rules implicitly depend on @com_google_protobuf_cc//:cc_toolchain,
-# which is the C++ proto runtime (base classes and common utilities).
+# java_lite_proto_library rules implicitly depend on @com_google_protobuf_javalite//:javalite_toolchain,
```
Comment (jin) [inline _posts/2017-02-27-protocol-buffers.md]: Backticks here too

File: _posts/2017-02-27-protocol-buffers.md [hunk 5]
```diff
@@ -36,37 +36,32 @@ Bazel's proto rules implicitly depend on the
...
 The following satisfies these dependencies:
 
-> TIP: This is a shortened version of
-> [https://github.com/cgrushko/proto_library/blob/master/WORKSPACE](https://github.com/cgrushko/proto_library/blob/master/WORKSPACE)
+> TIP: Clone https://github.com/cgrushko/proto_library/ to try protobufs in Bazel now.
 
 ```python
-# proto_library rules implicitly depend on @com_google_protobuf//:protoc,
-# which is the proto-compiler.
+# proto_library, cc_proto_library, and java_proto_library rules implicitly
```
Comment (jin) [inline _posts/2017-02-27-protocol-buffers.md]: Surround the rule and target names in backticks?

File: _posts/2017-02-27-protocol-buffers.md (hunk 1 above)
Comment (cgrushko) [reply _posts/2017-02-27-protocol-buffers.md]: GitHub automatically linkifies links - see preview in https://github.com/bazelbuild/bazel-blog/blob/80146308bdfef262d2b1d7102de75af03b6e5992/_posts/2017-02-27-protocol-buffers.md

File: _posts/2017-02-27-protocol-buffers.md (hunk 2 above)
Comment (cgrushko) [reply _posts/2017-02-27-protocol-buffers.md]: GitHub automatically linkifies links - see preview in https://github.com/bazelbuild/bazel-blog/blob/80146308bdfef262d2b1d7102de75af03b6e5992/_posts/2017-02-27-protocol-buffers.md

File: _posts/2017-02-27-protocol-buffers.md (hunk 4 above)
Comment (cgrushko) [reply _posts/2017-02-27-protocol-buffers.md]: These are inside a comment inside a code block - is it useful? they won't look different, and the backticks will be in the final result.

File: _posts/2017-02-27-protocol-buffers.md (hunk 5 above)
Comment (cgrushko) [reply _posts/2017-02-27-protocol-buffers.md]: These are inside a comment inside a code block - is it useful? they won't look different, and the backticks will be in the final result.

Comment (jin) [reply _posts/2017-02-27-protocol-buffers.md]: I missed it, sorry.

File: _posts/2017-02-27-protocol-buffers.md (hunk 1 above)
Comment (jin) [reply _posts/2017-02-27-protocol-buffers.md]: This will be converted to HTML for the bazel blog, so we'll need the []() syntax

**Comment by cgrushko** [comment]
Good point about the target not being GitHub :) Done.

File: _posts/2017-02-27-protocol-buffers.md (hunk 1 above)
Comment (cgrushko) [reply _posts/2017-02-27-protocol-buffers.md]: Good point, done.

**Pull request merged**